packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx

```diff
@@ -534,71 +545,96 @@ const BottomSheetActionListTemplate: ComponentStory<typeof BottomSheetComponent>
-
-export const WithHeader = WithHeaderTemplate.bind({});
-
-const WithFooterTemplate: ComponentStory<typeof BottomSheetComponent> = () => {
-  const [isOpen, setIsOpen] = React.useState(true);
-
-  return (
-    <BaseBox>
-      <Button onClick={() => setIsOpen(true)}>Add address</Button>
-      <BottomSheetComponent
-        isOpen={isOpen}
-        onDismiss={() => {
-          setIsOpen(false);
-        }}
-      >
-        <BottomSheetHeader title="Saved Address" />
+      <BottomSheetComponent isOpen={isOpen2} onDismiss={() => setIsOpen2(false)}>
+        <BottomSheetHeader />
         <BottomSheetBody>
-          <BaseBox padding="spacing.4">
-            <RadioGroup label="Addresses">
-              <Radio value="home">Home - 11850 Florida 24, Cedar Key, Florida</Radio>
-              <Radio value="office">Office - 2033 Florida 21, Cedar Key, Florida</Radio>
-            </RadioGroup>
-          </BaseBox>
+          <RadioGroup label="Addresses">
+            <Radio value="home">Home - 11850 Florida 24, Cedar Key, Florida</Radio>
+            <Radio value="office">Office - 2033 Florida 21, Cedar Key, Florida</Radio>
+          </RadioGroup>
+          <Button isFullWidth onClick={() => setIsOpen2(false)}>
+            Add Another
+          </Button>
         </BottomSheetBody>
-        <BottomSheetFooter
-          trailing={{ primary: { text: 'Add address' }, secondary: { text: 'Remove address' } }}
-        />
+        <BottomSheetFooter>
+          <Button isFullWidth variant="secondary">
+            Remove address
+          </Button>
+          <Button isFullWidth marginTop="spacing.5">
+            Add address
+          </Button>
+        </BottomSheetFooter>
       </BottomSheetComponent>
     </BaseBox>
   );
 };
 
-export const WithFooter = WithFooterTemplate.bind({});
+export const WithHeader = WithHeaderTemplate.bind({});
+
+// const WithFooterTemplate: ComponentStory<typeof BottomSheetComponent> = () => {
+//   const [isOpen, setIsOpen] = React.useState(true);
+
+//   return (
+//     <BaseBox>
+//       <Button onClick={() => setIsOpen(true)}>Add address</Button>
+//       <BottomSheetComponent
+//         isOpen={isOpen}
+//         onDismiss={() => {
+//           setIsOpen(false);
+//         }}
+//       >
+//         <BottomSheetHeader title="Saved Address" />
+//         <BottomSheetBody>
+//           <BaseBox padding="spacing.5">
+//             <RadioGroup label="Addresses">
+//               <Radio value="home">Home - 11850 Florida 24, Cedar Key, Florida</Radio>
+//               <Radio value="office">Office - 2033 Florida 21, Cedar Key, Florida</Radio>
+//             </RadioGroup>
+//           </BaseBox>
+//         </BottomSheetBody>
+//         <BottomSheetFooter
+//           trailing={{ primary: { text: 'Add address' }, secondary: { text: 'Remove address' } }}
+//         />
+//       </BottomSheetComponent>
+//     </BaseBox>
+//   );
+// };
+
+// export const WithFooter = WithFooterTemplate.bind({});
```

Comment:
Let's remove this comment block if not needed

Reply:
Right.